File: DSA with Python/Arrays/find_duplicates.py
def find_duplicates(arr,k):

    duplicate_elements = []

    for i in range(0,k+1):
        for j in range(i+1,k+1):
            if arr[i] == arr[j]:
                duplicate_elements.append(arr[i])

    return duplicate_elements

if __name__ == "__main__":
    # defining a list
    list1 = []

    # take the input from user regarding
    n = int(input('Enter number of elements you want in the list : '))

    for i in range(0, n+1):
        num = int(input(f'Enter element number {i} : '))
        list1.append(num)

    print(f'The original list entered by user : {list1}')

    duplicates = find_duplicates(list1,n)

    print(f'Duplicate elements in list are : {duplicates}')

File: DSA with C++/Arrays/find_duplicates.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
    vector<int> arr;
    int n, num, i, j;

    cout<<"Enter the size of the string : ";
    cin>>n;

    for(i=0;i<n;i++)
    {
        cout<<"Enter the element number "<<i<<" :";
        cin>>num;

        arr.push_back(num);
    }

    cout<<"\nThe array given by user : ";
    for(i=0;i<n;i++)
    {cout<<arr[i]<<" ";}

    cout<<"\nDuplicate Elements in the array : ";

    for(i=0;i<n;i++)
    {
        for(j=i+1;j<n;j++)
        {
            if(arr[i]==arr[j])
            {
                cout<<arr[i]<<" ";
            }
        }
    }


    return 0;
}